utils/order-utils/test/orders.js

```diff
@@ -27,12 +27,13 @@ describe('Orders', async () => {
   const rinkebySwap = '0x43f18D371f388ABE40b9dDaac44D1C9c9185a078'
 
   const NOV_7_2020_22_18_14 = '1604787494'
+  const MAY_11_2017_00_00_00 = '1494460800'
 
   orders.setVerifyingContract(rinkebySwap)
 
   it('Checks that a generated order is valid', async () => {
     const order = await orders.getOrder({
-      expiry: '1494460800',
+      expiry: MAY_11_2017_00_00_00,
       nonce: '101',
```

Comment:
can nonce be pulled out as well to be a const?

Reply:
Yes, very happy to. Just hoping for a bit more discussion around this as I feel like its an undefined rule of when we pull out a const and when we don't.

I see that pulling out the timestamp above provides way more clarity on the variable - what was previously a random looking string of numbers is now a clearly defined date and so it can be seen that it is may 11th etc. Similar clarity is gained by pulling out an address and replacing it with `ASTAddress`.

I'm just wondering if pulling out nonce gives any more clarity than it already has as is? Or is the reason you're asking for this is because the same number is used many times? And should I also be pulling out the `param: x` numbers as const too?